### PR TITLE
autossh: add Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by Magrid0 — APPROVED** [review]
looks good to me, thanks for your contribution!

File: pages.es/common/autossh.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# autossh
+
+> Ejecuta, monitoriza y reinicia conexiones SSH.
```
Comment (navarroaxel) [inline pages.es/common/autossh.md:3]:
```suggestion
> Ejecuta, monitorea y reinicia conexiones SSH.
```

File: pages.es/common/autossh.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+> Auto-reconecta para mantener los túneles de reenvío de puertos. Acepta todas las señales `ssh`.
+> Más información: <https://www.harding.motd.ca/autossh>.
+
+- Inicia una sesión SSH, reiniciando cuando un puerto de monitorización no devuelve datos:
```
Comment (navarroaxel) [inline pages.es/common/autossh.md:7]:
```suggestion
- Inicia una sesión SSH, reiniciando cuando un puerto de monitoreo no retorna datos:
```

File: pages.es/common/autossh.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+`autossh -M {{puerto_monitor}} "{{comando_ssh}}"`
+
+- Reenvía un puerto local a uno remoto, reiniciando cuando sea necesario:
+
+`autossh -M {{puerto_monitor}} -L {{puerto_local}}:localhost:{{puerto_remoto}} {{usuario}}@{{host}}`
+
+- Fork `autossh` en segundo plano antes de ejecutar `ssh` y no abre un shell remoto:
```
Comment (navarroaxel) [inline pages.es/common/autossh.md:15]:
```suggestion
- Crea un proceso `autossh` en segundo plano antes de ejecutar `ssh` y no abre un shell remoto:
```

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**